src/test/java/org/jboss/windup/rules/tests/WindupRulesTest.java

```diff
@@ -288,6 +288,7 @@ private void runWindup(GraphContext context, File baseRuleDirectory, final List<
         windupConfiguration.addDefaultUserRulesDirectory(baseRuleDirectory.toPath());
 
         final String baseRulesPathNormalized = baseRuleDirectory.toPath().normalize().toAbsolutePath().toString();
+        System.out.println("baseRulesPathNormalized AAA " + baseRulesPathNormalized);
```

Comment:
I am fine with this being logged, but could you format it a little better? :) I'm sure this was just debugging code that was left in, but I think it could be useful anyway.